docs/source/model-registry.rst

```diff
@@ -152,6 +152,60 @@ While the method above creates an empty registered model with no version associa
         run_id="d16076a3ec534311817565e6527539c0"
     )
 
+Fetching an MLflow Model from the Model Registry
+------------------------------------------------
+
+After you have registered an MLflow model, you can fetch that model using :meth:`~mlflow.pyfunc.load_model`. 
```

Comment:
i think in the code example below using `mlflow.pyfunc.load_model` is fine, but here we should say `mlflow.<model_flavor>.load_model` to match `mlflow.<model_flavor>.log_model` above (line 92).

Reply:
Might be worth to mention, there are two ways to fetch a model: `<model_flavor>.load_model` and `mlflow.pyfunc.load_model`. I think @andychow-db and I discussed it, but we felt it might confuse the developer. However, we do mention above that there are two ways to register the model:`mlflow.<model_flavor>.log_model()` and `mlflow.register_model()`. Likewise, it's not unreasonable to mention that there are two ways to fetch or retrieve a registered model. But focus on the example showing `mlflow.pyfunc.load_model`, because only the `mlflow.pyfunc.load_model` can be served using `mlflow serve models -m URI`

@sueann is that reasonable?

Other than that it LGTM.

Reply:
mm my comment was even simpler - just that we should be consistent in this paragraph with line 92. will add a suggestion here

Reply:
Yes, I was making an argument to mention both ways. And your simple suggestion below is spot on.